packages/augur-ui/src/modules/common/form.tsx

```diff
@@ -509,10 +510,20 @@ export class CategorySingleSelect extends Component<
 > {
   state: CategorySingleSelectState = {
     selected: this.props.initialSelected || '',
-    value: this.props.initialValue || this.props.initialSelected || '',
+    value: this.props.initialValue || (this.props.initialSelected === CUSTOM ? '' : this.props.initialSelected || ''),
     showText: this.props.initialSelected === CUSTOM,
   };
 
+  componentWillReceiveProps(nextProps) {
```

Comment:
we are starting to get away from legacy react methods, use `componentDidUpdate`

Reply:
ohhhhh interesting 👍